flake.nix

```diff
@@ -74,9 +74,9 @@
               runtimeInputs = [ config.packages.bech32 pkgs.jq pkgs.git pkgs.nodejs ];
               text = ''
                 [ -f assets/doom1.wad ] || curl https://distro.ibiblio.org/slitaz/sources/packages/d/doom1.wad -o assets/doom1.wad
-                ln -sf ${config.packages.doom-wasm}/websockets-doom.js assets/websockets-doom.js
-                ln -sf ${config.packages.doom-wasm}/websockets-doom.wasm assets/websockets-doom.wasm
-                ln -sf ${config.packages.doom-wasm}/websockets-doom.wasm.map assets/websockets-doom.wasm.map
+                [ -f assets/websockets-doom.js ] || ln -s ${config.packages.doom-wasm}/websockets-doom.js assets/websockets-doom.js
```

Comment:
could we do an env var instead? e.g. `USE_CUSTOM_ASSETS` so users always get the latest built assets by default?

Reply:
Good point.. I made this only work for my workflow. Need to fix it for default use.